ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -241,6 +242,29 @@ export const useMhrInformation = () => {
     return isDraft ? getMhrTransferHomeOwnerGroups.value : ownerGroups
   }
 
+  const parseDeletedWillOwnerGroups = (): any => {
```

Comment:
Return type should defined, looks like it is an array of owners...

Reply:
Changed new function return type as well as older ones